### PR TITLE
add kserve gpu tests

### PR DESCRIPTION
**Pull request by rohithkrn**
## Description

Add kserve gpu test workflow

### CODE REVIEW

File: .github/workflows/kserve_gpu_tests.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: KServe CPU Nightly Tests
```
Comment (agunapal) [inline .github/workflows/kserve_gpu_tests.yml:1]: GPU

File: .github/workflows/kserve_gpu_tests.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+on:
+  workflow_dispatch:
+  # runs everyday  at 5:15am
+  schedule:
+    - cron:  '15 5 * * *'
+
+jobs:
+  kserve-cpu-tests:
```
Comment (agunapal) [inline .github/workflows/kserve_gpu_tests.yml:10]: gpu

**Review by agunapal — CHANGES_REQUESTED** [review]
Can you please add a test to check that the GPU is actually being used.

File: .github/workflows/kserve_gpu_tests.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+on:
+  workflow_dispatch:
+  # runs everyday  at 5:15am
+  schedule:
+    - cron:  '15 5 * * *'
+
+jobs:
+  kserve-gpu-tests:
+    # runs-on: [self-hosted, regression-test-gpu
```
Comment (agunapal) [inline .github/workflows/kserve_gpu_tests.yml:11]: looks like this needs to be uncommented?

Comment (rohithkrn) [reply .github/workflows/kserve_gpu_tests.yml]: fixed

**Review by agunapal — APPROVED** [review]
LGTM

**Pull request merged**